python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```
@@ -619,6 +619,11 @@ keys:
         ip_address:
           type: str
           description: IPv4 address/mask or "dhcp".
+        ip_address_secondaries:
+          type: list
+          items:
+            type: str
+            description: IPv4 address/mask or "dhcp".
```

DHCP is not allowed as secondary.
```suggestion
            description: IPv4 address/mask.
```